packages/app/src/ResetPasswordPage.tsx

```diff
@@ -59,7 +57,7 @@ export function ResetPasswordPage(): JSX.Element {
             </Group>
           </Stack>
         )}
-        {success && <div data-testid="success">Email sent</div>}
+        {success && <div>If the account exists on our system, a password reset email will be sent.</div>}
```

Comment:
I think it would be okay to just leave this as a success message, no real need to qualify it for the sake of total honesty about our security measures

Reply:
Ha, true.  I took the recommended message verbatim from the pen test report.